Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: dev
+dev: bundle
+	bundle exec jekyll serve --host 0.0.0.0
+
+.PHONY: local-dev
+local-dev: bundle
+	bundle exec jekyll serve
+
+.PHONY: bundle
+bundle:
+	bundle install --path vendor/bundle
```

Comment:
Why not use `--deployment` here instead? http://bundler.io/v1.16/guides/using_bundler_in_applications.html#deployment